### ISSUE TITLE
Support bulk import of rights metadata

### ISSUE THREAD

**Issue by alexwlchan** [wellcomecollection/archivematica-infrastructure#114]
Currently users have to supply rights information by listing every single file as a new line in `rights.csv`, e.g.

```csv
file,basis,status,determination_date,jurisdiction,start_date,end_date,terms,citation,note,grant_act,grant_restriction,grant_start_date,grant_end_date,grant_note,doc_id_type,doc_id_value,doc_id_role
objects/Full_Presentation_Friday.ppt,license,,,,,,,,CC-BY,use,,,,Open,,,
objects/Full_presentation_Thurs.ppt,license,,,,,,,,CC-BY,use,,,,Open,,,
objects/Full_presentation_Wed.ppt,license,,,,,,,,CC-BY,use,,,,Open,,,
```

It should be possibly to specify rights for the entire transfer package as `objects/`, similar to what we do in `metadata.csv`.

**Comment by sevein** [comment]
After looking more into this, the work can have two different scopes. [Upstream issue #1340](https://github.com/archivematica/Issues/issues/1340), especially [this comment](https://github.com/archivematica/Issues/issues/1340#issuecomment-1138626196), describes rights at different levels: the complete transfer, a subdirectory, or one file. Rights on a more specific path should replace the broader rights. An [earlier upstream comment](https://github.com/archivematica/Issues/issues/1340#issuecomment-758043095) also suggests storing these as PREMIS 3 folder/package-level rights instead of repeating them for every file.

There are two possible options:

- **Support only `objects/`:** Archivematica would treat this row as transfer-level rights. Its existing METS generation would then write these rights against every original file. This is still a small change that solves this issue, but not the complete behaviour requested in #1340.

- **Support directory-level rights and overrides** Archivematica would support transfer, subdirectory and file scopes, including inheritance and more-specific overrides. A proper PREMIS 3 implementation would affect the CSV importer, rights scope modelling, transfer and AIP METS generation, reingest/backlog handling, and tests. This is a bigger change across several modules, and it needs a design investigation before we can give a reliable estimate.

The local rights validator also needs some care:

- It currently expects every `file` value to match one payload file, so it rejects directory paths.
- It would need to normalize and validate directory paths, including deciding whether values such as `objects/`, `/objects` and BagIt paths are accepted.
- It should not reject every combination of transfer-, directory- and file-level rows. The upstream requirement allows more-specific rights to override broader rights.
- We need clear conflict and precedence rules. Multiple rows at the same level can be valid, for example when describing different PREMIS grant acts.
- The Lambda and Archivematica must use the same path and override rules. Otherwise, the Lambda could accept a CSV that behaves differently during ingest.

The remaining decision is whether the smaller `objects/`-only solution is acceptable in the short term, or whether we should wait until we have capacity to implement the complete directory and override behaviour described in #1340.

**Comment by sevein** [comment]
After discussing this further, we confirmed that we don't need to support partial restrictions. Everything in the transfer should be either open or closed, so we can use the simpler `objects/`-only approach and don't need to support directory-level rights or overrides.

The implementation should be fairly small and split across two repositories:

- **`archivematica-infrastructure`:** update `verify_rights_csv_is_valid` so that the exact value `objects/` is accepted as a transfer-wide target without trying to resolve it as a payload file. Existing file-only CSVs can remain valid for backwards compatibility, but a CSV should not mix `objects/` with file-specific rows because partial rights are out of scope. Other directory paths remain unsupported. Add focused tests and update the transfer-package documentation and Lambda README.
- **Archivematica:** update `rights_from_csv.py` so that an `objects/` row creates one `RightsStatement` with the `Transfer` metadata-applies-to type and the transfer UUID. File rows should continue to create file-scoped statements as they do today. Add importer tests covering the transfer-scoped row.

Archivematica already does the rest: transfer METS and final AIP METS generation find transfer-scoped rights and write them against every original file, using the correct file linking identifier. Reingest consumes those METS rights rather than rerunning the CSV importer. We should add an explicit METS regression test, but we should not need schema, migration, workflow, API, METS production-code, directory modelling, inheritance, override, or reingest changes.

**Comment by sevein** [comment]
It has been confirmed in production that the solution works.
